cg/cli/upload/utils.py

```diff
@@ -12,16 +12,16 @@
 
 
 class LinkHelper:
```

Comment:
This class seem to handle statusDB models. Should we place it closer to statusDB as these can be used in more than upload?

Reply:
Good idea, I've moved it to `cg/store/api/link.py`